server/render/index.js

```diff
@@ -68,12 +67,18 @@ export function render( element, key = JSON.stringify( element ) ) {
 	//todo: render an error?
 }
 
-export function serverRender( context ) {
-	if ( config.isEnabled( 'server-side-rendering' ) ) {
+export function serverRender( req, res ) {
```

Comment:
Not new in this PR, but the name `serverRender` makes me think that this func will always render content.